src/main/webapp/app/exercises/shared/statistics/exercise-detail-statistics.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="row">
     <jhi-doughnut-chart
         class="col-md-4"
-        [course]="exercise.course!"
+        [course]="exercise.course ?? exercise.exerciseGroup!.exam!.course!"
```

Comment:
We have a helper method for exactly this behavior called `getCourseFromExercise`